transcom/mymove
# Int b 17980 prime counselor approve reject email

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a869619)

## Summary

This branch sends an email to the customer once the Prime has completed counseling for a move.

### How to test

**Note**: I was having trouble creating a shipment that allowed the Prime to click the "Complete Counseling" button, if you have a better workflow to get to that point feel free to try it/leave a comment. The method below relies on editing the database using DBeaver. The testharness move also doesn't assign an origin duty location, so this adds another step below.

1. Access the testharness scenarios page and select the "**HHGMoveWIthServiceItemsandPaymentRequestsForTIO"**. Be sure to note both the **move locator** and the **service member id.**
![Screenshot 2024-01-24 at 2 56 16 PM](https://github.com/transcom/mymove/assets/146959895/2c51a82a-80f0-4e90-91a5-6fb1387bfa58)
2. Find the service member using that id in the `service_members` table in DBeaver, edit the email to be your personal email. Be sure to click the "Save" button after editing.
![Screenshot 2024-01-24 at 2 56 53 PM](https://github.com/transcom/mymove/assets/146959895/5f2e8b9a-d256-4259-82b7-74acdbf28a58)
3. Login as a TIO, find move, go to "Payment Requests" tab, and click "View Orders"
![Screenshot 2024-01-24 at 3 04 03 PM](https://github.com/transcom/mymove/assets/146959895/ab5b25dc-94d9-4f2f-b96e-f170b3ab7c1c)
4. Edit the origin duty location to any actual location (i.e. Yuma AFB).
5. Login as Prime simulator, find move and click "Complete Counseling".
6. You should now receive an email similar to the following:
![Screenshot 2024-01-24 at 3 08 16 PM](https://github.com/transcom/mymove/assets/146959895/427a1181-60a9-441d-ba5c-0ddf60b66566)
